pom.xml

```diff
@@ -77,6 +77,16 @@ under the License.
       <id>repo.jenkins-ci.org</id>
       <url>https://repo.jenkins-ci.org/public/</url>
     </repository>
+    <repository>
+      <id>shib-release</id>
+      <url>https://build.shibboleth.net/nexus/content/groups/public</url>
```

Comment:
@ChrisRo89 Why? Is there any dependency available exclusively on this repository?

Reply:
`net.shibboleth.utilities`? not in Maven Central?

Reply:
opensaml >3.3x is not in the Maven Central, the latest version in Maven Central is 3.3.0 from 2017, it seems related to security reasons https://wiki.shibboleth.net/confluence/display/DEV/Use+of+Maven+Central